docs/source/guides/cache_system.mdx

```diff
@@ -156,15 +155,16 @@ usage: optimum-cli neuron cache [-h] {create,set,add,list} ...
 positional arguments:
   {create,set,add,list}
     create              Create a model repo on the Hugging Face Hub to store Neuron X compilation files.
-    set                 Set the name of the Neuron cache repo to use locally.
-    add                 Add a model to the cache of your choice.
-    list                List models in a cache repo.
+    set                 Set the name of the Neuron cache repo to use locally (trainium only).
+    add                 Add a model to the cache of your choice (trainium only).
+    list                List models in a cache repo (trainium only).
+    synchronize         Synchronize local compiler cache with the hub cache.
```

Comment:
Is it both for trainium and inferentia? Otherwise I would specify that it's inferentia only as well.

Reply:
I wasn't sure: it uploads everything under the directory corresponding to NEURON_COMPILE_CACHE_URL. I will put inferentia for now.

Reply:
Alright. 

Also, could you copy everything under a directory in the cache repo for now. I do not know how it will collide with the current system otherwise?

Reply:
It does not collide, because the neuronx cached files are automatically put under a root `neuronxcc-<version>` directory.